src/main/scala/cromwell/binding/types/WdlObjectType.scala

```diff
@@ -1,16 +1,30 @@
 package cromwell.binding.types
 
 import cromwell.binding.Call
-import cromwell.binding.values.{WdlCallOutputsObject, WdlObject}
+import cromwell.binding.values.{WdlCallOutputsObject, WdlMap, WdlObject, WdlString}
+
+import scala.util.{Failure, Success}
 
 case object WdlObjectType extends WdlType {
   val toWdlString: String = "Object"
 
   override protected def coercion = {
     case o: WdlObject => o
+    case m: WdlMap if m.wdlType.keyType == WdlStringType && WdlStringType.isCoerceableFrom(m.wdlType.valueType) =>
```

Comment:
I don't remember if this is covered elsewhere, but do we support coercing `JsObject` into `WdlObject`?